src/main/java/seedu/dictionote/logic/parser/CliSyntax.java

```diff
@@ -12,6 +12,9 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_CONTENT = new Prefix("c/");
+    public static final Prefix PREFIX_WEEK = new Prefix("/week");
+    public static final Prefix PREFIX_HEADER = new Prefix("/header");
+    public static final Prefix PREFIX_MAINCONTENT = new Prefix("/maincontent");
```

Comment:
mayble we should be consistent here, and use week/ header/ maincontent/, or w/ h/ mc/

Reply:
Agree, to be consistent with the other prefixes in the CliSyntax.java, I think w/, h/ and mc/ is better